tyxe/priors.py

```diff
@@ -139,6 +139,9 @@ def update_(self, net):
         for module_name, module in net.named_modules():
             for site_name, site in list(util.named_pyro_samples(module, recurse=False)):
                 full_name = module_name + "." + site_name
+                # See change in DictPrior as an alternative
```

Comment:
Solution below is preferable in my opinion. Comments to be removed after review.